card/db/migrate_core_cards/20161103154836_update_keys.rb

```diff
@@ -0,0 +1,11 @@
+# -*- encoding : utf-8 -*-
+
+class UpdateKeys < Card::Migration::Core
+  def up
+    Card.pluck(:id, :name, :key).each do |id, name, key|
+      new_key = name.to_key
+      next if new_key == key
+      Card.where(id: id).update_all(key: new_key)
```

Comment:
Technically, we should probably add something to update referee_keys in the card_references table.  The impact would be quite small, because the id is far more for existing cards, but it would be a good measure for data quality.